react/features/base/media/middleware.js

```diff
@@ -60,12 +60,15 @@ function _setRoom({ dispatch, getState }, next, action) {
     // Read the config.
 
     const state = getState();
+    const isRoomValid = Boolean(room);
```

Comment:
We actually have a function called `isRoomValid`! I wanted us to use it here but I guess I was vague in my comment. Anyway, I'll handle that.

Reply:
Oh, I misunderstood, sorry. Thanks for taking care of it!